books/projects/x86isa/machine/instructions/arith-and-logic.lisp

```diff
@@ -295,6 +295,107 @@
 
     x86))
 
+
+
+(def-inst x86-adcx/adox
+
+  :parents (three-byte-opcodes)
+
+  :short "Operand Fetch and Execute for ADD, ADC, SUB, SBB, OR, AND,
```

Comment:
This documentation seems to be copied from `x86-add/adc/sub/sbb/or/and/xor/cmp-G-E`. Updating the documentation is sufficient, but I'd suggest adding `gpr-adcx/adox-spec` to `gpr-arith/logic-spec` and then adding support for those instructions to `x86-add/adc/sub/sbb/or/and/xor/cmp-G-E`.

Reply:
Thanks, dumb copy-paste -- will fix the doc.  I actually tried doing what you suggested before defining a separate function for adox/adcx, but there were a few things that made me think adox/adcx didn't sufficiently fit the patterns assumed by that function. In particular, the operand size criteria for adox/adcx seems incompatible to me -- the x86-add/etc. function assumes the operand size is 1 if the low bit of the opcode is 0 (as in the code below), and it respects the 0x66 operand size override prefix which is a mandatory prefix for ADCX.
```
   (b* (...
       (byte-operand? (eql 0 (the (unsigned-byte 1)
                               (logand 1 opcode))))
       ((the (integer 1 8) operand-size)
        (select-operand-size
         proc-mode byte-operand? rex-byte nil prefixes nil nil nil x86))
```

Reply:
I see, that makes sense then.